### PR TITLE
Added editRaw option to fields

### PR DESCRIPTION
**Pull request by Nitwel**
grafik
grafik

### CODE REVIEW

**Comment by benhaynes** [comment]
Nice!! This is looking great! A few quick updates:

* "Edit raw" to "Raw value"
* Title-case for modal title (capitalize "Raw Value" and "Object")
* Use `code` icon in drop-down
* Move option to top of context menu
* Use  monospace textarea or code component

**Comment by rijkvanzanten** [comment]
> Title-case for modal title (capitalize "Raw Value" and "Object")

Lets use translations on the type instead of title formatting it

**Comment by rijkvanzanten** [comment]
@benhaynes Should this option be last in that dropdown list? Feels like the most "advanced" one of the bunch

**Comment by benhaynes** [comment]
Yeah, the end is fine!

File: app/src/components/v-form/form-field.vue [hunk 1]
```diff
@@ -33,15 +38,30 @@
 			@input="$emit('input', $event)"
 		/>
 
+		<v-modal :active="showRawModal" :title="$t('edit_raw_value')" :subtitle="$t(`js_types.${type}`)">
+			<v-textarea v-model="rawString" :placeholder="$t('enter_raw_value')"></v-textarea>
```
Comment (rijkvanzanten) [inline app/src/components/v-form/form-field.vue:42]:
```suggestion
			<v-textarea v-model="rawString" :placeholder="$t('enter_raw_value')" />
```

File: app/src/components/v-form/form-field.vue [hunk 2]
```diff
@@ -102,7 +122,69 @@ export default defineComponent({
...
+		const { showRawModal, rawString, editRaw, saveRaw, undoRaw, type } = useRaw();
+
+		return { isDisabled, marked, _value, editRaw, saveRaw, undoRaw, showRawModal, rawString, type };
+
+		function useRaw() {
+			const showRawModal = ref(false);
+			const rawString = ref('');
+
+			const type = computed(() => {
+				let type = typeof props.value || props.initialValue;
```
Comment (rijkvanzanten) [inline app/src/components/v-form/form-field.vue:134]: We should read the type from the field info, not the current value

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**